app.rb

```diff
@@ -69,7 +69,7 @@ def config
 
       idp_url = authorization_url(ial: ial, aal: params[:aal])
 
-      logger.info("Redirecting to #{idp_url}")
+      settings.logger.info("Redirecting to #{idp_url}") if ENV['RACK_ENV'] != 'test'
```

Comment:
Can you explain the changes here?

Reply:
I am making the logger call work; as it was, nothing was being logged. I made it conditional, to suppress the output when running the specs. Let me know if you have an opinion (of course).

Reply:
I like that idea (of globally configuring the logger conditionality). Thanks for that tip.